### ISSUE TITLE
Fails on debian wheezy

### ISSUE THREAD

**Issue by romanr** [geerlingguy/ansible-role-logstash-forwarder#3]
TASK: [geerlingguy.logstash-forwarder | Ensure logstash-forwarder is started and enabled on boot.] ***
failed: [website] => {"failed": true}
msg: update-rc.d: error: no runlevel symlinks to modify, aborting!

**Comment by romanr** [comment]
Manually:

```
# update-rc.d logstash-forwarder defaults
update-rc.d: using dependency based boot sequencing
insserv: script logstash-forwarder is not an executable regular file, skipped!


# ls-al /etc/init.d/logstash-forwarder
-rw-r-xr-x 1 root 2.7K Dec 17 22:52 /etc/init.d/logstash-forwarder

# chmod a+x /etc/init.d/logstash-forwarder
# ls -al  /etc/init.d/logstash-forwarder
-rwxr-xr-x 1 root 2.7K Dec 17 22:52 /etc/init.d/logstash-forwarder

# update-rc.d logstash-forwarder defaults
update-rc.d: using dependency based boot sequencing
```

Perhaps "mode" here is not correct:

```
- name: Install logstash-forwarder init script.
  copy:
    src: logstash-forwarder.deb.init
    dest: /etc/init.d/logstash-forwarder
    mode: 0655
```

**Comment by romanr** [comment]
should be mode: 0755

**Comment by geerlingguy** [comment]
This was fixed in #2 - thanks for confirming the problem!
